config/config-server/resources/cruise-config.xsd

```diff
@@ -880,7 +880,6 @@
         <xsd:sequence>
             <xsd:choice>
                 <xsd:element ref="baseTask" minOccurs="0" maxOccurs="1"/>
-                <xsd:any minOccurs="0" maxOccurs="1"/>
```

Comment:
According to the [documentation](https://www.go.cd/documentation/user/15.3.0/configuration/configuration_reference.html#oncancel)

> Specifies a task to execute when a stage is cancelled. Only one task can be defined in `<oncancel>`.

Why was this needed?